roxie/ccd/ccdstate.cpp

```diff
@@ -2925,6 +2925,11 @@ class CRoxiePackageSetManager : implements IRoxieQueryPackageManagerSet, impleme
                 soapTraceLevel = control->getPropInt("@level", 0);
                 topology->setPropInt("@soapTraceLevel", soapTraceLevel);
             }
+            else if (stricmp(queryName, "control:soapLogSepString")==0)
+            {
+                if (control->hasProp("@string"))
+                    control->getProp("@string", soapSepString);
```

Comment:
is this theoretically thread unsafe in combination with the placement of the code in HThorWSCBaseActivity::init() ?
e.g. if an activity is initializing when this thread is also initializing.

Also, if called multiple times soapSepString will grow (should be soapSepString.clear())